docker.local/config/sc.yaml

```diff
@@ -139,21 +139,19 @@ smart_contracts:
     # allocation settings for free storage
     # these values are applied to all free allocations
     free_allocation_settings:
-      # the free payment is split between read and write pool in the ratio
-      # read_pool_fraction : (1-read_pool_fraction)
-      read_pool_fraction: 0.2
-      data_shards: 10
-      parity_shards: 5
-      size: 10000000000
+      data_shards: 2
       duration: 50h
+      max_challenge_completion_time: 1m
```

Comment:
This max_challenge_completion_time is not being used, we can remove it.

Reply:
@peterlimg max_challenge_completion_time is the cct set. it's the cct set for all blobbers..

Reply:
yeah thanks, got it